airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/ConfigRepository.java

```diff
@@ -778,4 +781,12 @@ public void loadDataNoSecrets(final ConfigPersistence seedPersistenceWithoutSecr
     persistence.loadData(seedPersistenceWithoutSecrets);
   }
 
+  private Condition includeTombstones(final Field<Boolean> tombstoneField, final boolean includeTombstones) {
+    if (includeTombstones) {
+      return DSL.trueCondition();
+    } else {
+      return tombstoneField.eq(false);
+    }
+  }
+
```

Comment:
this one doesn't fit in this PR thematically but I wanted to use this convenience method in multiple upcoming PRs